packages/database/src/core/view/ChildChangeAccumulator.ts

```diff
@@ -41,44 +34,50 @@ export class ChildChangeAccumulator {
       childKey !== '.priority',
       'Only non-priority child changes can be tracked.'
     );
-    const oldChange = safeGet(this.changeMap_, childKey) as Change;
+    const oldChange = this.changeMap.get(childKey);
```

Comment:
@mikelehen Do you know if there is a reason for `safeGet()`?. Are we guarding against queries for `length` or something similar?

Reply:
I think it's just guarding against accidentally getting a value from the prototype chain.